changelog/63050.added

```diff
@@ -0,0 +1 @@
+netapi_enable_clients option to allow enabling/disabling of clients in salt-api
```

Comment:
The file extension for the file should be `.changed`, as this will require users to make a change to their configs.

Please elaborate here that this will require all users using the Salt-API to define which clients they want to use with the salt-api.

Reply:
Done